File: 8kyu/(8 kyu) Square(n) Sum/(8 kyu) Square(n) Sum.cs
public static class Kata
{
  public static int SquareSum(int[] n)
  {
    int res = 0;
    
    for (int i = 0; i < n.Length; i++)
    {
      res += n[i] * n[i];
    }
    
    return res;
  }
}

File: 8kyu/(8 kyu) Square(n) Sum/(8 kyu) Square(n) Sum.go
package kata

// #1
// func SquareSum(numbers []int) int {
//   res := 0
//   for i := 0; i < len(numbers); i++ {
//     res += numbers[i] * numbers[i]
//   }  
//   return res
// }
// #2
// func SquareSum(numbers []int) (res int) {
//     for _, n := range numbers {
//        res = res + n*n
//     }
//     return
// }
// #3
func SquareSum(numbers []int) (res int) {
  for _, n := range numbers { res += n*n }
  return
}
